cmd/odfcli/root.go

```diff
@@ -0,0 +1,96 @@
+package odfcli
```

Comment:
I got the suggestion from @sp98  and this directory helps in collaborating and expanding commands and easier to collaborate when multiple are working on the same sub-command. Also, this is a common directory structure seen in other downstream cli tools.

@sp98 would you like to add anything?

Reply:
If we follow the structure similar to kubectl as @BlaineEXE pointed out, I'd suggest `root.go` in `cmd/`, and we have subdirectories such as `set`, `get`, etc.

Reply:
we can't keep `main.go` and `root.go` in under same directory, it will import cycle issue.  What we can do is keep `main.go` in root(similar to go.mod) directory and keep `root.go` under `cmd/` I have seen this directory structure also.

Reply:
How about this?
```
cmd/main.go
cmd/root/root.go
cmd/get/...
cmd/set/...

Reply:
A common pattern that is used follows `cmd/appname`  where `appname` is the name of the executable. 

https://github.com/golang-standards/project-layout/tree/master/cmd

Reply:
Going with this structure 
```
├── cmd
│   ├── main.go
│   ├── odfcli
│   │   ├── get
│   │   │   ├── get.go
│   │   │   └── get_recovery_profile.go
│   │   └── set
│   │       ├── set.go
│   │       └── set_recovery_profile.go
│   └── root
│       └── root.go
```

Reply:
@sp98 Nice link for the go standards, makes sense to have subdirectories to easily support multiple binaries. Not sure if we'll ever have another binary in this repo, but doesn't hurt to allow for it with this pattern. With that pattern then, seems like we need this structure so that all the go files for the binary are in its folder, and the folder is named `odf` like the tool:

```
├── cmd
│   ├── odf
│   |   ├── main.go
│   │   ├── get
│   │   │   ├── get.go
│   │   │   └── get_recovery_profile.go
│   │   └── set
│   │       ├── set.go
│   │       └── set_recovery_profile.go
│   |   └── root
│   |      └── root.go
```

Reply:
done